src/ngraph/runtime/gpu/cuda_emitter.cpp

```diff
@@ -280,6 +281,79 @@ size_t runtime::gpu::CUDAEmitter::build_1d_max_pool(const GPURuntimeContext* ctx
     return primitive_index;
 }
 
+size_t runtime::gpu::CUDAEmitter::build_elementwise_n_to_1(const GPURuntimeContext* ctx,
```

Comment:
what's the `n_to_1` mean here?

Reply:
That it handles unary/binary/ternary/.../n input elementwise operations where there is only a single output.